source/read.txt

```diff
@@ -88,8 +81,8 @@ The following example returns the number of documents in the specified collectio
    :copyable:
    :dedent:
 
-.. TODO: To learn more about the ``count_documents()`` method, see the
-.. :ref:`kotlin-sync-accurate-count` guide.
+To learn more about the ``countDocuments()`` method, see the
+:ref:`kotlin-sync-accurate-count` guide.
```

Comment:
S: since you're linking to a section within the Count Documents guide, I'd reword to something like:
```suggestion
:ref:`kotlin-sync-accurate-count` section of the Count Documents guide.
```

Reply:
Applies to the two links below as well